src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -557,9 +574,31 @@ void RenderTarget::applyCurrentView()
 {
     // Set the viewport
     const IntRect viewport = getViewport(m_view);
-    const int     top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
+    int           top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
```

Comment:
I think it would be more readable to keep `top` const here and make another const int for scissor top instead of reusing this variable. Also naming them `viewportTop` and `scissorTop` respectively.